### PR TITLE
content for 2025-01-17-doom

### PR DESCRIPTION
**Pull request by ben-mor**

### CODE REVIEW

File: website/content/events/2025-01-17-doom/_index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: "Probability of Doom"
+date: 2025-12-18T10:24:50+01:00
+eventStart: 2025-01-17T18:00:00+01:00
+eventEnd: 2025-01-17T20:30:00+01:00
+eventLocation: "Veranstaltungsraum, Haus des Engagements, Rehlingstraße 9, 79100 Freiburg"
+eventGeoLat: 47.98953
+eventGeoLon: 7.83979
+meetupLink: https://www.meetup.com/de-DE/rationality-freiburg/
+lwLink: https://www.lesswrong.com/groups/fFZZ2Ywzsab86EESY
+description: "Where: Haus des Engagements, Rehlingstraße 9, 79100 Freiburg. When: Friday, December 20th 2024 at 18:00 hours CET."
```
Comment (omarkohl) [inline website/content/events/2025-01-17-doom/_index.md]: Date wrong. I know it's annoying to set it in two places :-(

File: website/content/events/2025-01-17-doom/_index.de.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+title: "Wahrscheinlichkeit des Untergangs"
+date: 2025-12-18T10:24:50+01:00
+eventStart: 2025-01-17T18:00:00+01:00
+eventEnd: 2025-01-17T20:30:00+01:00
+eventLocation: "Veranstaltungsraum, Haus des Engagements, Rehlingstraße 9, 79100 Freiburg"
+eventGeoLat: 47.98953
+eventGeoLon: 7.83979
+meetupLink: https://www.meetup.com/de-DE/rationality-freiburg/
+lwLink: https://www.lesswrong.com/groups/fFZZ2Ywzsab86EESY
+description: "Wo: Haus des Engagements, Rehlingstraße 9, 79100 Freiburg. Wann: Freitag, 20. Dezember 2024 um 18:00 Uhr MEZ."
```
Comment (omarkohl) [inline website/content/events/2025-01-17-doom/_index.de.md]: See other comment

File: website/content/events/2025-01-17-doom/_index.de.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+#build:
+#  list: never
+---
+
+
+## Vorbereitung
+
+Dieser Event basiert auf Ideen die ich hier gelernt habe:
+
+[ACX - The Extinction Tournament](https://www.astralcodexten.com/p/the-extinction-tournament)
```
Comment (omarkohl) [inline website/content/events/2025-01-17-doom/_index.de.md]: optional: highlight that the link is in English e.g by appending "(Englisch)".

File: website/content/events/2025-01-17-doom/_index.de.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Am Ende kommen wir wieder zusammen und tauschen uns darüber aus, ob und was wir als nächstes tun sollten.
+
+## Organisation
+
+Mach dir keine Sorgen, wenn du denkst, dass du nichts beitragen kannst! Jeder ist willkommen!
+
+Es gibt immer eine Mischung aus Deutsch- und Englischsprachigen, und wir gestalten die Diskussionsrunden so, dass sich jeder wohlfühlt. Die Hauptsprache ist Englisch.
+
+Dieses Treffen wird von Omar moderiert.
```
Comment (omarkohl) [inline website/content/events/2025-01-17-doom/_index.de.md]: Ben

**Review by omarkohl — CHANGES_REQUESTED** [review]
almost perfect :slightly_smiling_face:

**Comment by ben-mor** [comment]
@omarkohl I integrated all your comments.

**Review by omarkohl — APPROVED** [review]
lgtm

**Pull request merged**